overlays/46-audio-receiver/usr/lib/systemd/system/uxplay.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=UxPlay, an AirPlay-Mirror and AirPlay-Audio server
+Documentation=https://github.com/FDH2/UxPlay
+After=network.target
+
+[Service]
+User=neon
```

Comment:
This should also come from config in the yml recipe (just a note but this user is hard-coded in other places already)

Reply:
@NeonDaniel I don't see an example of how to template an overlay file using a variable. Would you please point me to one or share the docs for doing so?

Reply:
I thought I had one but I don't, so resolving this as a "later" problem to figure out templating for overlays